conda_lock/conda_lock.py

```diff
@@ -256,8 +269,8 @@ def make_lock_files(
             for pkg in link_actions:
                 url = fetch_by_dist_name[pkg["dist_name"]]["url"]
                 md5 = fetch_by_dist_name[pkg["dist_name"]]["md5"]
-                r = requests.head(url, allow_redirects=True)
-                url = r.url
+                # r = requests.head(url, allow_redirects=True)
```

Comment:
This will make it faster and we don't need to check the URLs. Maybe just remove these lines?